pom.xml

```diff
@@ -16,6 +16,8 @@
   <modules>
     <module>immutables-exceptions</module>
     <module>hubspot-style</module>
+    <module>immutable-collection-encoding</module>
```

Comment:
Should the name be plural, eg `encodings`? All the other artifact names (`immutables`, `immutables-exceptions`) seem to plural